scripts/docker-utils.sh

```
@@ -0,0 +1,46 @@
+
+  echo "\nBuilding image"
+  docker build -t $image_name .
+}
+
+build_docker_container() {
+  local image_name="$1"
+  local container_name="$2"
+  local host_port="$3"
+  local container_port="$4"
+
+  echo "\nBuilding container"
```

Shellcheck (0.10.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)